src/components/message/message.stories.mdx

```diff
@@ -120,6 +123,36 @@ Message variant is `info` by default
   </Story>
 </Preview>
 
+### With rich content
+It is also possible to use the `List`, `ListItem` and `Typography` components to render content other than plain text in a `Message`.
+<Preview>
+  <Story name="with list content">
```

Comment:
Title is `rich content` but story name is `list content`. Shouldn't these be the same?

Reply:
Yeah good spot, I decided to make the title more general but forgot to update the name